### PR TITLE
Meeting 2024-01-11 minutes draft

### PR DESCRIPTION
**Pull request by sthagen**
- [x] pre meeting draft
- [x] post meeting draft

### CODE REVIEW

**Review by stacywray — APPROVED** [review]

File: meeting_minutes/240111_SARIF_TC_83.md [hunk 1]
```diff
@@ -90,44 +93,88 @@ The minutes were ...
...
 
 ## 3.2 Review outcomes of subgroup discussions
 
-* ...
+* Stefan: Reports from first editorial meeting since the last TC meeting
 
 ## 3.3 Discuss the list of small non-breaking changes for SARIF v2.2
 
- * ... 
+ * Aditiya: On precision and severity topic two changes require further detailing which he will do together with Micahel before the next meeting
```
Comment (stacywray) [inline meeting_minutes/240111_SARIF_TC_83.md:125]: correct Michael spelling

File: meeting_minutes/240111_SARIF_TC_83.md [hunk 2]
```diff
@@ -90,44 +93,88 @@ The minutes were ...
...
 
 ## 3.4 Review Roadmap [Future.md](https://github.com/oasis-tcs/sarif-spec/blob/main/Future.md)
 
-* ...
+* All reviwe the pull request [futures addition #621](https://github.com/oasis-tcs/sarif-spec/pull/621)
```
Comment (stacywray) [inline meeting_minutes/240111_SARIF_TC_83.md:129]: Correct "review" spelling

File: meeting_minutes/240111_SARIF_TC_83.md [hunk 3]
```diff
@@ -90,44 +93,88 @@ The minutes were ...
...
+* David Keaton: Reports of discussions he remembers when GitLab assessed if they want to integrate SARIF in their platform
+* David Malcolm: Has it been decided to widen the scope?
+* David Keaton: 
+  * The TC already decided that SARIF v3.0 will widen and include entries from the topical list present in the future document
+  * Suggests to continue the discussion on the meaning of "S" in SARIF within [issue #603](https://github.com/oasis-tcs/sarif-spec/issues/603)
+* Charles: Introduces the second change block of the two proposed changes on the future document concerning profiling of the schema
+* Stefan: 
+  * Is in favor of adding that. 
+  * The implementation is challenging, but widening the scope of SARIF will 
+    * enlarge reagions which are of interest only to some use cases and would
```
Comment (stacywray) [inline meeting_minutes/240111_SARIF_TC_83.md:143]: Correct "regions" spelling

File: meeting_minutes/240111_SARIF_TC_83.md [hunk 4]
```diff
@@ -90,44 +93,88 @@ The minutes were ...
...
+* DECISION to merge the changed section "Provide Domain-specific Schema Subsets" of [futures addition #621](https://github.com/oasis-tcs/sarif-spec/pull/621) be into the future.md main revision
 
 ## 5.3 Review of Action Items (Co-Chair David)
 
-* Ongoing ACTION (from former meetings) ...
-* ACTION on ...
+* Ongoing ACTIONS (from former meetings):
+  * ACTION on Michael to triage old issues
+  * ACTION on Michael to organize label renovation
+* ACTION on Davoid Malcolm to create a first link list for SARIF producers and consumers (from the wikipedia article draft on SARIF)
```
Comment (stacywray) [inline meeting_minutes/240111_SARIF_TC_83.md:175]: Correct "David" spelling

File: meeting_minutes/240111_SARIF_TC_83.md [hunk 5]
```diff
@@ -90,44 +93,88 @@ The minutes were ...
...
+* ACTION on Charles to start the list of links to SARIF related activities on the TC repo wiki
```
Comment (stacywray) [inline meeting_minutes/240111_SARIF_TC_83.md:176]: Add "hyphen" between "SARIF" "related"

**Review by stacywray — CHANGES_REQUESTED** [review]
Commented on minor spelling errors.

**Review by dmk42 — APPROVED** [review]
Thanks!

**Pull request merged**